test/connect-cloud-health/ConnectHealth.test.js

```diff
@@ -184,26 +194,26 @@ describe('Connect Cloud Health test suite', function () {
             }
         };
         let cloudHealth = new index_1.HealthChecker();
-        cloudHealth.registerReadinessCheck(
+        cloudHealth.registerStartupCheck(
         // tslint:disable-next-line:no-shadowed-variable
-        new index_1.ReadinessCheck("startup", new Promise(function (resolve, reject) {
+        new index_1.StartupCheck("startup", () => new Promise(function (resolve, reject) {
             resolve();
         })));
         index_1.ReadinessEndpoint(cloudHealth)(reqStub, resStub, nextStub);
     });
-    it('Readiness returns 200 OK and UP on readiness and liveness checks', function (done) {
+    it('Readiness returns 200 OK and UP on startup and liveness checks', function (done) {
```

Comment:
should this be "startup and readiness checks"?

Reply:
Done